bucket/aws-iam-authenticator.json

```
@@ -0,0 +1,16 @@
+{
+    "homepage": "https://docs.aws.amazon.com/eks/latest/userguide/install-aws-iam-authenticator.html",
+    "description": "A tool to use AWS IAM credentials to authenticate to a Kubernetes cluster",
+    "license": "Apache-2.0",
+    "version": "2018-12-06",
+    "url": "https://amazon-eks.s3-us-west-2.amazonaws.com/1.11.5/2018-12-06/bin/windows/amd64/aws-iam-authenticator.exe",
+    "hash": "4ca7b218c0dec3cb1a396ca6f206bc882071918a6599e6c24ecca486d1fec30d",
+    "bin": "aws-iam-authenticator.exe",
+    "checkver": "<b>Windows</b>: <a href=\"https://amazon-eks.s3-us-west-2.amazonaws.com/(?<kubernetes>[\\d.]+)/(?<version>[\\d-]+)/bin/windows/amd64/aws-iam-authenticator.exe\"",
```

I think it could be simplified a bit. If they change layout this should be still viable, unless they change URL.

```suggestion
    "checkver": "/(?<kubernetes>[\\d.]+)/(?<version>[\\d-]+)/bin/windows/amd64/aws-iam-authenticator.exe\"",
```